openedx/core/djangoapps/profile_images/views.py

```diff
@@ -17,9 +17,12 @@
     OAuth2AuthenticationAllowInactiveUser,
     SessionAuthenticationAllowInactiveUser,
 )
+from openedx.core.lib.api.parsers import TypedFileUploadParser
 from openedx.core.lib.api.permissions import IsUserInUrl, IsUserInUrlOrStaff
 from openedx.core.djangoapps.user_api.accounts.image_helpers import get_profile_image_names, set_has_profile_image
-from .images import validate_uploaded_image, create_profile_images, remove_profile_images, ImageValidationError
+from .images import (
```

Comment:
Ned emailed out a follow-up amendment to the email, saying that leading dots follow the convention used when importing the module that contains them, and so are okay. 

Quote:

    A correction: Sven Marnach challenged my statement below that 
    relative imports should be avoided, and he is correct.  The new recommendation is:

        from lms.djangoapps.hologram.models import 3DExam    # GOOD

        from .models import 3DExam                           # GOOD

        from hologram.models import 3DExam                   # BAD!

    Keep in mind that relative imports (starting with a dot) will piggy-back 
    on how the importing module itself was imported, so the third form 
    above is still no good.

    I've updated the Python guidelines again, including more detail 
    about other places to be careful, like patching, urls.py, and settings.py: 
    https://github.com/edx/edx-platform/wiki/Python-Guidelines#imports